tests/load/c/execute.go

```diff
@@ -53,13 +49,6 @@ func execute(ctx context.Context, preFundedKeys []*secp256k1.PrivateKey, config
 		return fmt.Errorf("ensuring minimum funds: %w", err)
 	}
 
-	registry := prometheus.NewRegistry()
```

Comment:
I don't think we really want to share the same registry for each execute call, each load test should be separated ideally 🤔